src/cmd/cmd_replay.ml

```diff
@@ -48,11 +49,14 @@ let run_file ~unsafe ~optimize ~entry_point ~invoke_with_symbols filename model
 
     let symbol_bool = symbol_char
 
-    let abort () = assert false
+    let abort () = ()
```

Comment:
I'm not sure about this because the execution will keep going on which would not be the case in the symbolic case.

We should probably display an error when reaching such a case.